src/System.Collections/src/System/Collections/Generic/HashSetEqualityComparer.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace System.Collections.Generic
+{
+
+    /// <summary>
+    /// Equality comparer for hashsets of hashsets
+    /// </summary>
+    /// <typeparam name="T"></typeparam>
+    internal class HashSetEqualityComparer<T> : IEqualityComparer<HashSet<T>>
+    {
+
+        private IEqualityComparer<T> m_comparer;
+
+        public HashSetEqualityComparer()
+        {
+            m_comparer = EqualityComparer<T>.Default;
+        }
+
+        public HashSetEqualityComparer(IEqualityComparer<T> comparer)
```

Comment:
Is this constructor used anywhere? I know it exists in the full framework, but looks like dead code that can be deleted.